### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by remy** [stevemonks/ZXSpectrumNextMachineCodeGuides#1]
I'm sure this all boils down to assembler flavour: I'm on a mac and don't have zcl.exe to hand. So I've appropriate adjustments to the .asm file so sjasmplus can read it (without errors), but it's this line that's confused me the most:

https://github.com/stevemonks/ZXSpectrumNextMachineCodeGuides/blob/master/asm/PlayerSprite/playersprite.s#L153

>     in a,(bc)       ; read in [P][O][I][U][Y]

Or in fact anywhere where you're doing `in a, (bc)` - since, as far as I can see (and I'm extremely new to fully groking asm - so please be gentle), there's no opcode for loading BC's reference into A (and sjasmplus is breaking here too).

Is Zeus doing something clever here that I've missed?

Thanks in advance.

**Comment by stevemonks** [comment]
I don't think it's anything clever, just Zeuss being forgiving and allowing in a,(bc) which is non standard but (to me at least) more descriptive of what's going on. Replacing these with in a,(c) should fix that. I should probably fix it up in the code to make it more universal.

**Comment by remy** [comment]
That's what I did in the code too and it got past this point. I suspected it might be exactly what you described.

There wasn't much to change in the .asm file too - mostly just adding flags and options for sjasmplus to compile to .nex correctly. I've pasted it here if you're interested at all https://gist.github.com/remy/efe26ba437a89745d5dd5d14cdc229af

Going a bit off topic, but checking the .snx version, I get the same bordered but with your mouse sprite in the top left.

My build version doesn't have the mouse, so, and uneducated guess says that I'm not uploading the sprite properly - or something's off in my build.
